runtime/call_metrics.go

```diff
@@ -234,3 +243,54 @@ func NewOutboundTChannelMetrics(scope tally.Scope) *OutboundTChannelMetrics {
 	metrics.SystemErrors = scope.Counter(outboundCallsSystemErrors)
 	return &metrics
 }
+
+// NewOutboundCustomMetrics returns outbound metrics for custom clients
+func NewOutboundCustomMetrics(scope tally.Scope) *OutboundCustomMetrics {
```

Comment:
Custom metrics do not belong here. Metrics integration belongs at the http client or tchannel client level.